src/Nethermind/Nethermind.Merge.Plugin/Data/ExecutionPayloadV3.cs

```diff
@@ -51,4 +52,12 @@ public override bool TryGetBlock(out Block? block, UInt256? totalDifficulty = nu
     /// </summary>
     [JsonRequired]
     public override ulong? ExcessBlobGas { get; set; }
+
+
+    /// <summary>
+    /// Gets or sets <see cref="Block.ParentBeaconBlockRoot"/> as defined in
+    /// <see href="https://eips.ethereum.org/EIPS/eip-4788">EIP-4788</see>.
```

Comment:
not 4788

Reply:
is this payload version same as V3 or V4? (also this comment is on the ParentBeaconBlockRoot which is Eip4788)